### PR TITLE
Feature/css structure

### PR DESCRIPTION
**Pull request by shrocket**
Please have a look at my CSS, including (but not limited to): 
- folder and file structure
- BEM implementation

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+.answer-button {
+  display: flex;
+  border: solid 1px var(--color-gray);
+  background-color: var(--color-scarlet);
+  border-radius: 5px;
+  color: var(--color-cream);
+  margin-bottom: 5px;
+  width: 100px;
+  font-size: 18px;
+}
```
Comment (TobiasSteinhagen) [inline components/card.css:47]: Just a suggestion, you could add an extra css-file for the question and the answer button as well.

**Review by TobiasSteinhagen — COMMENTED** [review]
Overall LGTM, just a little suggestion :)

File: components/card.css (hunk 1 above)
Comment (shrocket) [reply components/card.css:47]: I didn't fully understand that structure then. A *whole* file for each element?

Comment (TobiasSteinhagen) [reply components/card.css:47]: You could do that, I just checked the handout again (wasn't sure anymore) and the suggested structure looks like this :

[https://github.com/neuefische/bo-web-23-4/blob/main/sessions/css-structure/challenges-css-structure.md#separate-the-quiz-app-css-code-into-different-files](url)

Hope that helps ✌️

Comment (shrocket) [reply components/card.css:47]: the link doesn't work?

**Pull request merged**